tests/unit/vdom.ts

```diff
@@ -2708,4 +2709,38 @@ describe('vdom', () => {
 			});
 		});
 	});
+
+	describe('behavior', () => {
+		it('will append nodes with attributes already attached', (test) => {
+			if (!global.window || !global.window.hasOwnProperty('MutationObserver')) {
+				test.skip('MutationObserver not present');
+			}
+
+			class Foo extends WidgetBase {
```

Comment:
Don’t think we need a widget to exercise this functionality do we? Any reason we can’t just use dom.create?